<launch>
  <!-- Start the desired image saving node for photometric visual servoing -->  
  <node name="pgmvsCaptureAndSaveDesired" pkg="ros_dvs_bridge" type="ros_dvs_bridge_pgmvsCaptureAndSaveDesired" output="screen"/> <!-- launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "> -->
    <param name="cameraTopic" type="string" value="/insta360/right/image_raw"/>
    <param name="logs" type="string" value="$(env HOME)/.ros/ros_dvs_bridge/"/>
 
    <param name="rosbagForEVS" type="bool" value="true"/>
    <param name="camera2tool" type="string" value="$(find ros_dvs_bridge)/config/insta360toTool.yml"/>
    <param name="cameraPoseTopic" type="string" value="/tf"/>
    <param name="desiredPoseTopicForEVS" type="string" value="/vs/camera/desired_pose"/>

</launch>

